ui/helpers/utils/util.js

```diff
@@ -39,8 +39,8 @@ export function formatDate(date, format = "M/d/y 'at' T") {
 
 export function formatDateWithYearContext(
   date,
-  formatThisYear = 'MMM d',
-  fallback = 'MMM d, y',
+  formatThisYear = "MMM d 'at' h:mm a",
+  fallback = "MMM d, y 'at' h:mm a",
```

Comment:
I think we can create another function for this format in that case we would not have to deal with test failure and updating the hook. wdyt?